### PR TITLE
T2D-213 Add phenotype registry api

### PR DESCRIPTION
**Pull request by Zhicheng-Liu**

### CODE REVIEW

File: src/main/java/uk/ac/ebi/ampt2d/registry/entities/Phenotype.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    @Id
+    @Column(nullable = false, unique = true, updatable = false)
+    private String id;
+
+    @ApiModelProperty(position = 3)
+    @JsonProperty
+    @NotNull
+    @Enumerated(EnumType.STRING)
+    @Column(nullable = false)
+    private Group phenotypeGroup;
```
Comment (selvaebi) [inline src/main/java/uk/ac/ebi/ampt2d/registry/entities/Phenotype.java:63]: just group instead of phenotypeGroup?

Comment (Zhicheng-Liu) [reply src/main/java/uk/ac/ebi/ampt2d/registry/entities/Phenotype.java:63]: No, "group" is a reserved word for a lot of databases.

Comment (selvaebi) [reply src/main/java/uk/ac/ebi/ampt2d/registry/entities/Phenotype.java:63]: ya i understand 👍

**Comment by selvaebi** [comment]
Looks fine to me , but we need to change the repo name or add a new repo ?

**Review by selvaebi — APPROVED** [review]

**Comment by selvaebi** [comment]
Need to add tag for Phenotype Entity in SwaggerConfiguration. Need to change title and description as well.

**Comment by Zhicheng-Liu** [comment]
Added in 9acd6a5. Forgot to include that in previous commit.

**Pull request merged**